docs/examples.rst

```diff
@@ -337,7 +337,7 @@ Fetch various data about the current state of the ERC20 contract.
     >>> erc20.functions.decimals().call()
     0
     >>> erc20.functions.totalSupply().call()
-    2040
+    2044
```

Comment:
@carver, I did this change just to pass doctest checks, not sure what caused that. Please check.

Reply:
@rayrapetyan it's the contract reading from live data (and new unicorn tokens being minted)